plugins/datamonitor/test/tst_pluginloader.cpp

```diff
@@ -0,0 +1,113 @@
+#include "qpluginloader.h"
+
+#include <QList>
+#include <QTest>
+
+#include <pluginbase/plugin.h>
+
+using namespace scopy;
+
+class TST_BareMinimum : public QObject
+{
+	Q_OBJECT
+private Q_SLOTS:
+	void fileExists();
+	void isLibrary();
+	void loaded();
+	void className();
+	void instanceNotNull();
+	void multipleInstances();
+	void qobjectcast_to_plugin();
+	void clone();
+	void name();
+	void metadata();
+};
+
+#define PLUGIN_LOCATION "../../plugins"
+#define FILENAME PLUGIN_LOCATION "/libscopy-bareminimumplugin.so"
+
+void TST_BareMinimum::fileExists()
```

Comment:
This test is outdated. It should use your own plugin .so and should be named accordingly.